pkg/service/endpoints_controller.go

```diff
@@ -56,10 +56,10 @@ func (e *EndpointController) SyncServiceEndpoints() error {
 			continue
 		}
 
-		glog.V(3).Infof("About to update endpoints for service %v", service.Name)
+		glog.V(5).Infof("About to update endpoints for service %s/%s", service.Namespace, service.Name)
```

Comment:
I'm thinking maybe meta.Identify(runtime.Object) or something.  We need to be able to crack the object efficiently and work for runtime.Unknown as well.

----- Original Message -----
> > @@ -56,10 +56,10 @@ func (e *EndpointController) SyncServiceEndpoints()
> > error {
> >  			continue
> >  		}
> >  
> > -		glog.V(3).Infof("About to update endpoints for service %v",
> > service.Name)
> > +		glog.V(5).Infof("About to update endpoints for service %s/%s",
> > service.Namespace, service.Name)
> 
> Maybe write a util.JoinName(namespace, name string) string function?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/3729/files#r23407127